### ISSUE TITLE
Native Ads plugin broken?

### ISSUE THREAD

**Issue by Nyankoo** [googleads/googleads-mobile-unity#2009]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2020.3.29f1
  * Google Mobile Ads Unity plugin version: 6.1.2 & Native (latest, no version number available)
  * Platform: Android
  * Platform OS version: Android 10
  * Any specific devices issue occurs on: _____
  * Mediation ad networks used, and their versions: _____

#### Steps to reproduce:

When following the docs on how to include the Native Ads plugin, multiple parts in your code can't be found/are missing, resulting in errors.

When also importing the normal version of the plugin (so both the normal and the native plugin are in the project), most errors go away, but all Native Ads methods from the docs are not found (eg. `OnUnifiedNativeAdLoaded`).

Changing them to their "normal" versions (eg. `OnUnifiedNativeAdLoaded `-> `OnNativeAdLoaded`) resolves the errors, but when calling  `adLoader.LoadAd(new AdRequest.Builder().Build());`, the following exception is thrown:

```
Exception: Error: called non-existent method System.String GoogleMobileAds.Api.AdRequest::BuildVersionString(System.String)
at GoogleMobileAds.Api.AdRequest.BuildVersionString (System.String )
at GoogleMobileAds.Android.AdLoaderClient.LoadAd (GoogleMobileAds.Api.AdRequest request)
```

So the question is, how to implement the Native Ads plugin correctly, or is this a bug?

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by Nyankoo** [comment]
@NVentimiglia Any insight on this? This issue blocks us from using AdMob.

**Comment by BauRawson** [comment]
I too encountered this issue (or a similar one?) and been watching this post, waiting for a solution.

- Unity version 2021.2.13f1
- Google Mobile Ads 7.0.0 & Google Native Ads plugin

When importing JUST the Native Ads package, I get this errors:

> Assembly 'Assets/GoogleMobileAdsNative/GoogleMobileAdsNative.dll' will not be loaded due to errors:
> Unable to resolve reference 'GoogleMobileAds'. Is the assembly missing or incompatible with the current platform?
> Assembly 'Assets/GoogleMobileAdsNative/GoogleMobileAdsNative.Android.dll' will not be loaded due to errors:
> Unable to resolve reference 'GoogleMobileAds.Common'. Is the assembly missing or incompatible with the current platform?
> Assembly 'Assets/GoogleMobileAdsNative/GoogleMobileAdsNative.Common.dll' will not be loaded due to errors:
> Unable to resolve reference 'GoogleMobileAds.Common'. Is the assembly missing or incompatible with the current platform?

I tried adding the External Dependency Manager package as well, but the errors persist.
And then I tried adding Google Mobile Ads 7.0.0, where I encountered OP's issue.


Sorry if this isn't the way to +1 on GitHub, and if this is adding noise instead of information let me know and I can delete it :)

**Comment by Nyankoo** [comment]
@NVentimiglia This is a really big issue, we need an update on this please!

**Comment by NVentimiglia** [comment]
The native ads plugin has been updated to v. 7.0.0. Please update it here. 

https://developers.google.com/admob/unity/native

**Comment by pfay-fc** [comment]
Does Google Mobile Ads Unity Plugin 6.1.2 still work with the older native ads plugin? And if so, where can you get a copy of the older native ads plugin?

**Comment by Nyankoo** [comment]
@NVentimiglia Thank you, but we would like to keep using 6.1.2 for now. Is there a fix available too?

**Comment by Nyankoo** [comment]
@NVentimiglia Please re-open! The issue wasn't fixed for version 6.1.2!

**Comment by Nyankoo** [comment]
@NVentimiglia Please re-open this issue or release a fix for either version 6.1.2, or the totally broken 7.0.0 release.
